internal/controller/indexer/indexer.go

```diff
@@ -71,6 +72,7 @@ func SetupIndexer(mgr ctrl.Manager) error {
 		setupApisixTlsIndexer,
 		setupApisixConsumerIndexer,
 		setupGatewayClassIndexer,
+		setupTLSHostIndexer,
```

Comment:
I suggest building indexes by resource type, which is more conducive to unified management.